web/client/components/background/BackgroundDialog.jsx

```diff
@@ -100,12 +104,12 @@ export default class BackgroundDialog extends React.Component {
 
     constructor(props) {
         super(props);
-        const pickedProps = pick(this.props, 'title', 'format', 'style', 'thumbnail');
+        const pickedProps = pick(this.props, 'title', 'format', 'style', 'thumbnail', 'attribution');
```

Comment:
WMS layers are currently supporting the property `credits` instead of attribution. `credits` is an object that provides separated information related to the attribution and this behaviour seems related to the WMS getCapabilities parsing see parsing [here](https://github.com/geosolutions-it/MapStore2/blob/master/web/client/api/WMS.js#L51-L64).

An example of `credits` property:

```js
// information parsed from the WMS capabilities
{
    title: "content of <Title></Title>",
    imageUrl: "url of the image linked as LogoURL",
    link: "http://some.site.of.reference",
    logo: { // more info about the logo
        format: "image/png",
        width: "200",
        height: "100"
    }
}
```

These information are used by the WMS layers in different ways:

- cesium https://github.com/geosolutions-it/MapStore2/blob/master/web/client/components/map/cesium/plugins/WMSLayer.js#L94

```js
const cr = options.credits;
const credit = cr ? new Cesium.Credit(cr.text || cr.title, cr.imageUrl, cr.link) : options.attribution;
```

- leaflet https://github.com/geosolutions-it/MapStore2/blob/master/web/client/components/map/leaflet/plugins/WMSLayer.js#L168

```js
attribution: options.credits && creditsToAttribution(options.credits),
```

- openlayers https://github.com/geosolutions-it/MapStore2/blob/master/web/client/components/map/openlayers/plugins/WMSLayer.js#L190


```js
const toOLAttributions = credits => credits && creditsToAttribution(credits) || undefined;
```

Note: the common function `creditsToAttribution` is converting the content of the object to a string see https://github.com/geosolutions-it/MapStore2/blob/master/web/client/utils/LayersUtils.js#L699-L703

```js
export const creditsToAttribution = ({ imageUrl, link, title }) => {
    // TODO: check if format is valid for an img (svg, for instance, may not work)
    const html = imageUrl ? `<img src="${imageUrl}" ${title ? `title="${title}"` : ``}>` : title;
    return link && html ? `<a href="${link}" target="_blank">${html}</a>` : html;
};
```

Looking at the code seems that a credits with the only property title it will work even if with an html text 

```js
// credits
{
    title: "my <a>layer</a>"
}
```

The proposal is to use the `credits` property and apply same behavior of WMS to all WMTS layer and use the title property as html field in the editor

Reply:
This is resolved, I replaced the property `attribution: 'string' ` with `credits: {title: string,  . . .}`